modules/apps/portal-workflow/portal-workflow-task-web/src/main/resources/META-INF/resources/view_content.jsp

```diff
@@ -65,6 +65,7 @@ renderResponse.setTitle(assetRenderer.getTitle(workflowTaskDisplayContext.getTas
 						assetEntry="<%= assetEntry %>"
 						assetRenderer="<%= assetRenderer %>"
 						assetRendererFactory="<%= assetRendererFactory %>"
+						showExtraInfo="<%= true %>"
```

Comment:
@ricardocousolr I'm a little concerned that this change might affect other use cases.. For example, other places that are expecting `false`..
Is it possible to make this more specific, like [here](https://github.com/liferay/liferay-portal/blob/045dfcd7cf0f5630008e30b11e4694570adea96a/modules/apps/asset/asset-publisher-web/src/main/java/com/liferay/asset/publisher/web/internal/display/context/AssetPublisherDisplayContext.java#L1942-L1951)? In our case, you could create a new method on `WorkflowTaskDisplayContext.java` where the default value is `false` and D&M would be responsible for changing this value.

Reply:
Thanks for your review @carolmariaabb! I'll implement those changes and update the pr.

Reply:
Hi again @carolmariaabb, 

I've pushed a new commit ([9e4549a](https://github.com/liferay-workflow/liferay-portal/pull/872/commits/9e4549aee7cd1e8c711c0275364526b5c936a508)) with the changes. Please let me know if I that was what you meant or if I should do any changes. 

Thanks!